### PR TITLE
Implement Netflix Clone

### PR DESCRIPTION
**Pull request by Yaser-123**
The goal of this project is to create a functional clone of the Netflix platform. This clone should mimic the core features of Netflix, including user authentication, browsing of movie/TV show catalogs, streaming video content, and user-specific recommendations.

Key Features:
1.Header Section
2.Hero Section
3.Content Sections
4.Footer Section

Technical Stack:
1.Frontend
2.Responsive Design
3.Consistent Theme

Additional Notes:
Collaborate with the design team to ensure visual fidelity.
Ensure cross-browser compatibility (Chrome, Firefox, Safari, Edge).
Solicit feedback from team members and potential users to refine the design.

https://github.com/dhairyagothi/50_days_50_web_project/assets/140265777/aa70757c-1047-4a1b-ba83-2fddc20d58c9

https://github.com/dhairyagothi/50_days_50_web_project/assets/140265777/fb63ee10-6ebe-4a6f-ba74-ddb8cdcc8f0d

### CODE REVIEW

**Comment by Yaser-123** [comment]
Issue #102

File: public/index.html [hunk 1]
```diff
@@ -238,7 +238,6 @@
             <div class="project-name">Scroll Dark Game</div>
             <a href="Scroll Game Dark Run/index.html" class="demo-link">Demo</a>
         </div>
-        <div class="table">
             <div class="day-number">Day 42</div>
             <div class="project-name">Amazon App</div>
             <a href="Amazon_Clone/index.html" class="demo-link">Demo</a>
```
Comment (dhairyagothi) [inline public/index.html]: fix this 
image

pull main branch first

**Review by dhairyagothi — CHANGES_REQUESTED** [review]

File: public/index.html (hunk 1 above)
Comment (dhairyagothi) [reply public/index.html]: @Yaser-123

**Comment by dhairyagothi** [comment]
image

resolve this

**Comment by Yaser-123** [comment]
Done bro resolved pr it

**Comment by anshul23102** [comment]
Closing - being refiled with proper detailed professional template per framework standards.

**Pull request merged**